### ISSUE TITLE
Unknown location

### ISSUE THREAD

**Issue by francoisthire** [Deducteam/lambdapi#81]
I wrote something wrong, but the error reported by lambdapi prints an unknown location while I think it should not.

```
symbol const U : TYPE

// Interpretation function (for data types as types).

symbol injective T : U ⇒ TYPE

definition reflexive : ∀a, T a ⇒ T a ⇒ T a ≔ λ A R, ∀ x, R x x
```

```
[unknown location] Sort expected, [?11[A,R,x,x]] infered.
```

**Comment by rlepigre** [comment]
Partially fixed by d7e29a79f5cf45226092a62feb3c0e09e1d7c822.

**Comment by fblanqui** [comment]
The error message is now:
`[tests/KO/81.lp, 7:0-62] Term [λ(A:?0), λ(R:?1[A]), ∀(x:?2[A,R]), R x x] does not have type [∀(a:U), T a ⇒ T a ⇒ T a].
`
I think that it is clear enough now.
